src/Disp.js

```diff
@@ -1996,10 +1996,10 @@ CM.Disp.Tooltip = function(type, name) {
 		}
 		else if (Game.buyMode == -1) {
 			if (Game.buyBulk == -1) {
-				l('tooltip').innerHTML = l('tooltip').innerHTML.split(Beautify(Game.Objects[name].getPrice())).join('-' + Beautify(CM.Sim.BuildingSell(Game.Objects[name], Game.Objects[name].basePrice, Game.Objects[name].amount, Game.Objects[name].free, Game.Objects[name].amount, 0)));
+				l('tooltip').innerHTML = l('tooltip').innerHTML.split(Beautify(Game.Objects[name].getPrice())).join('-' + Beautify(CM.Sim.BuildingSell(Game.Objects[name], Game.Objects[name].basePrice, Game.Objects[name].amount, Game.Objects[name].free, Game.Objects[name].amount)));
 			}
 			else {
-				l('tooltip').innerHTML = l('tooltip').innerHTML.split(Beautify(Game.Objects[name].getPrice())).join('-' + Beautify(CM.Sim.BuildingSell(Game.Objects[name], Game.Objects[name].basePrice, Game.Objects[name].amount, Game.Objects[name].free, Game.buyBulk, 0)));
+				l('tooltip').innerHTML = l('tooltip').innerHTML.split(Beautify(Game.Objects[name].getPrice())).join('-' + Beautify(CM.Sim.BuildingSell(Game.Objects[name], Game.Objects[name].basePrice, Game.Objects[name].amount, Game.Objects[name].free, Game.buyBulk)));
 			}
```

Comment:
The `CM.Sim.BuildingSell` method signature has been changed, leading to these modifications.
It's the only place where the `CM.Sim.BuildingSell` is called other than the chocolate egg calculation one.

**I've noticed that this isn't actually doing anything**, as `Beautify(Game.Objects[name].getPrice())` is never matched in the string; maybe this is some legacy code to remove?

Reply:
I'll repeat what I wrote in Issue #359 :

It's the return string of the function that is being matched. This is still needed.

Notice that Beautify(Game.Objects[name].getPrice()) is not in quotes. You won't find it in the source CC. It's a dynamic string.

It is to change the value of the tooltip to my calculated value.  If I'm misunderstanding you, please correct me.